.test-cafe/install_npm.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+# https://discuss.circleci.com/t/how-to-change-node-version-in-circleci-2-0/17455?gclid=CjwKCAjwxt_tBRAXEiwAENY8hcOzhf0QTBne8qSSrslqR6OixN0xNJIGarf6jsz2Siwgs1SYMPNSqhoCGIgQAvD_BwE
+touch $BASH_ENV  
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
+echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4397703)